File: _write_exceptional.js
const fs = require("fs");
const path = "D:/03_AITOOL/deer-flow/frontend/src/app/workspace/graph-exceptional-field-theory/page.tsx";

const Q = String.fromCharCode(34);
const BT = String.fromCharCode(96);
const D = String.fromCharCode(36);
const LB = String.fromCharCode(123);
const RB = String.fromCharCode(125);

let c = [];

function q(s) { return Q + s + Q; }
function bt(s) { return BT + s + BT; }
function jsx(s) { return LB + s + RB; }

// Will build content
console.log("Script ready");

File: _write_exceptional.py
import os

target = r"D:_AITOOL\deer-flowrontend\srcpp\workspace\graph-exceptional-field-theory\page.tsx"

# Read the base64 data from companion file
import base64
with open(r"D:_AITOOL\deer-flow\_exceptional_b64.txt", "r") as f:
    b64data = f.read().strip()

content = base64.b64decode(b64data).decode("utf-8")
with open(target, "w", encoding="utf-8") as f:
    f.write(content)

print(f"Written {len(content)} bytes to {target}")
